sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/README.md

```diff
@@ -0,0 +1,278 @@
+# Azure Authoring client library for .NET
+
+Azure Conversations Authoring is part of the Azure Cognitive Service for Language, a cloud-based service that provides tools for creating, managing, and deploying conversational AI solutions. This client library offers the following features:
+
+* Creating and managing conversation projects
+* Importing and exporting conversation projects
+* Training models for conversational AI
+* Evaluating trained models
+* Deploying conversational AI models
+* Swapping deployments for active models
+* Canceling active training jobs
+* Managing project snapshots
+* Deleting trained models and deployments
+
+[Source code][source_root] | [Package (NuGet)][package]| [API reference documentation][text_refdocs] | [Product documentation][text_docs] | [Samples][source_samples]
+
+## Getting started
+
+This section should include everything a developer needs to do to install and create their first client connection *very quickly*.
+
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/ ):
+
+```dotnetcli
+dotnet add package Azure.AI.Language.Conversations.Authoring --prerelease
+```
+
+|SDK version  |Supported API version of service
+|-------------|-----------------------------------------------------
+|1.0.0-beta.1 | 2022-05-01, 2023-04-01, 2023-11-15-preview (default)
```

Comment:
Newest API version 2024-11-15-preview?

Reply:
Yes, this API version should be included in supported APIs

Reply:
Bidisha, thanks for your comments. Have updated now.